### PR TITLE
watch kube config and update model & tree (#24)

### LINKED ISSUE #24: Track changes in ~/.kube/config and update resource tree
The plugin should update it's resource tree upon changes done to ~/.kube/config via kubectl, oc, etc.

### PR DESCRIPTION
**Pull request by adietish**
fixes #24

### CODE REVIEW

**Comment by adietish** [comment]
@jeffmaury, @lstocchi: please review

File: src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/Contexts.kt [hunk 1]
```diff
@@ -73,10 +76,36 @@ class Contexts(
...
+		clear(false)
+		observable.fireModified(this) // invalidates root bcs there's no tree node for this
+	}
+
+	private fun closeCurrent() {
+		closeCurrent(true)
+	}
+
+	private fun closeCurrent(notify: Boolean) {
+		if (current == null) {
```
Comment (lstocchi) [inline src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/Contexts.kt:100]: Is this check necessary? If this.current is null, its null value will be given to `val current` then the `return` will be made at `current?.close() ?: return`, no?

**Review by lstocchi — APPROVED** [review]
LGTM. Tested on CRC and it worked.

File: src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/Contexts.kt (hunk 1 above)
Comment (adietish) [reply src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/Contexts.kt:100]: great catch, thx!

**Pull request merged**